pwndbg/commands/search.py

```diff
@@ -126,8 +126,19 @@ def search(type, hex, string, executable, writable, value, mapping, save, next):
 
     # Null-terminate strings
     elif type == 'string':
+        value = value.encode()
         value += b'\x00'
 
+    # Find the mappings that we're looking for
+    mappings = pwndbg.vmmap.get()
+
+    if mapping:
+        mappings = [m for m in mappings if mapping in m.objfile]
```

Comment:
`mapping` and `mappings` seems confusing to me...

Also below there is `pwndbg.search.search` called with `mapping=mapping` keyword argument and you have changed that command to get `mappings` keyword argument (NOTE: additional `s`). This is the reason the code is failing atm.